code/__DEFINES/hud.dm

```diff
@@ -26,183 +26,183 @@
 	"1:2,3:4" is the square (1,3) with pixel offsets (+2, +4); slightly right and slightly above the turf grid.
 	Pixel offsets are used so you don't perfectly hide the turf under them, that would be crappy.
 
-	In addition, the keywords NORTH, SOUTH, EAST, WEST and CENTER can be used to represent their respective
-	screen borders. NORTH-1, for example, is the row just below the upper edge. Useful if you want your
+	In addition, the keywords TOP, BOTTOM, RIGHT, LEFT and CENTER can be used to represent their respective
+	screen borders. TOP-1, for example, is the row just below the upper edge. Useful if you want your
 	UI to scale with screen size.
 
 	The size of the user's screen is defined by client.view (indirectly by world.view), in our case "15x15".
 	Therefore, the top right corner (except during admin shenanigans) is at "15,15"
 */
 
-/proc/ui_hand_position(i) //values based on old hand ui positions (CENTER:-/+16,SOUTH:5)
+/proc/ui_hand_position(i) //values based on old hand ui positions (CENTER:-/+16,BOTTOM:5)
 	var/x_off = i % 2 ? 0 : -1
 	var/y_off = round((i-1) / 2)
-	return"CENTER+[x_off]:16,SOUTH+[y_off]:5"
+	return"CENTER+[x_off]:16,BOTTOM+[y_off]:5"
 
 /proc/ui_equip_position(mob/M)
-	var/y_off = round((M.held_items.len-1) / 2) //values based on old equip ui position (CENTER: +/-16,SOUTH+1:5)
-	return "CENTER:-16,SOUTH+[y_off+1]:5"
+	var/y_off = round((M.held_items.len-1) / 2) //values based on old equip ui position (CENTER: +/-16,BOTTOM+1:5)
+	return "CENTER:-16,BOTTOM+[y_off+1]:5"
 
-/proc/ui_swaphand_position(mob/M, which = 1) //values based on old swaphand ui positions (CENTER: +/-16,SOUTH+1:5)
+/proc/ui_swaphand_position(mob/M, which = 1) //values based on old swaphand ui positions (CENTER: +/-16,BOTTOM+1:5)
 	var/x_off = which == 1 ? -1 : 0
 	var/y_off = round((M.held_items.len-1) / 2)
-	return "CENTER+[x_off]:16,SOUTH+[y_off+1]:5"
+	return "CENTER+[x_off]:16,BOTTOM+[y_off+1]:5"
 
 //Lower left, persistent menu
-#define ui_inventory "WEST:6,SOUTH:5"
+#define ui_inventory "LEFT:6,BOTTOM:5"
```

Comment:
What exactly is going on with the directional stuff here?

Reply:
https://user-images.githubusercontent.com/77420409/170252933-42cd08d6-ef86-47f1-83f3-59e2cc5a9dfa.mp4

Basically, it is so this works. In this video, I don't do that change and you see how the UI elements do not scale.